src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,30 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private double bladeWidth;
+    private double weight;
```

Comment:
The fields 'bladeWidth' and 'weight' should have access modifiers to encapsulate the properties of the Bulldozer class. Consider using 'private' to restrict access to these fields and provide getters and setters for manipulation.